services/workflows-service/src/data-analytics/data-analytics.service.ts

```diff
@@ -67,6 +68,113 @@ export class DataAnalyticsService {
     throw new Error(`No evaluation function found for rule name: ${(inlineRule as InlineRule).id}`);
   }
 
+  async checkMerchantOngoingAlert(
+    businessReport: BusinessReport,
+    {
+      increaseRiskScorePercentage,
+      increaseRiskScore,
+      maxRiskScoreThreshold,
+    }: CheckRiskScoreOptions,
+    alertSeverity: AlertSeverity,
+  ) {
+    const { report, businessId, projectId } = businessReport;
+
+    if (
+      !(
+        report as {
+          data: {
+            previousReport?: unknown;
+          };
+        }
+      ).data.previousReport
+    ) {
+      return;
+    }
+
+    const {
+      data: {
+        summary: { riskScore: currentRiskScore },
+        previousReport: {
+          summary: { riskScore: previousRiskScore },
+          reportType: previousReportType,
+        },
+      },
+    } = report as {
+      data: {
+        summary: {
+          riskScore: number;
+        };
+        previousReport: {
+          summary: {
+            riskScore: number;
+          };
+          reportType: BusinessReportType;
+        };
+      };
+    };
+
+    if (previousReportType !== BusinessReportType.ONGOING_MERCHANT_REPORT_T1) {
+      this.logger.warn(`Previous report type is not ONGOING_MERCHANT_REPORT_T1`);
+
+      return;
+    }
+
+    if (currentRiskScore < previousRiskScore) {
```

Comment:
i think we should add log that describe what company is it and what are the scores and an explanation that we skipped

Reply:
```      this.logger.warn(`Risk score skip a Business: ${businessId} Project: ${projectId} `, {
        currentRiskScore,
        previousRiskScore
      });

Reply:
it's not an issue, it's the correct functionality in this check

Reply:
the check is in place, i just asked to add a info log ...